scripts/remove_scsi.py

```diff
@@ -0,0 +1,54 @@
+import sys
+import os
+
+
+def main(work_directory):
+    list_of_files = list()
+    for (dir_path, dir_names, file_names) in os.walk(work_directory):
+        list_of_files += [os.path.join(dir_path, file) for file in file_names]
+    for file in list_of_files:
+        if file.endswith(".scs"):
+            is_file_processed = False
+            buffer_file_list = []
+            with open(file, 'r', encoding='utf-8') as scs_file:
+                dir_path = os.path.split(file)[0]
+                for scs_file_line in scs_file:
+                    if scs_file_line.find('[*^"file://') >= 0:

```

Comment:
Is it possible for multiple `scsi` file links to be on the same line? If so, this won't work for all of them.